source/utilities.tex

```diff
@@ -114,6 +114,22 @@
   template<class... T>
     using index_sequence_for = make_index_sequence<sizeof...(T)>;
 
+  template<class T> struct tuple_size;
+  template<size_t I, class T> struct tuple_element;
+
+  // \ref{intseq.binding} structured binding support for \tcode{integer_sequence}
+  template<class T, T... Values>
+    struct tuple_size<integer_sequence<T, Values...>>;
+
+  template<size_t I, class T, T... Values>
+    struct tuple_element<I, integer_sequence<T, Values...>>;
+
+  template<size_t I, class T, T... Values>
+    struct tuple_element<I, const integer_sequence<T, Values...>>;
```

Comment:
The paper does not have a blank line in the synopsis, and I'm not convinced one should be added. See also my previous comment.